test/e2e/create_test.go

```diff
@@ -143,6 +143,29 @@ func (tc *testContext) testBYOHConfiguration(t *testing.T) {
 	require.NoError(t, err, "failed to create Windows MachineSet")
 	machines, err := tc.waitForWindowsMachines(int(gc.numberOfBYOHNodes), "Provisioned", false)
 	require.NoError(t, err, "Machines did not reach expected state")
+
+	// Change the default shell to Powershell for the first BYOH VM. We expect there to be BYOH VMs with either
+	// cmd or powershell as the default shell, so getting a mix of both, either between different BYOH VMs, or with the
+	// VMs spun up for the Machine testing is important.
+
+	// This needs to be retried as it will only succeed once the VM is fully initialized
+	// This should not increase the overall test time, as WMCO would have to wait for the VM to be ready anyway.
+	err = wait.Poll(nodeRetryInterval, time.Minute*10, func() (done bool, err error) {
+		command := "New-ItemProperty -Path \"HKLM:\\SOFTWARE\\OpenSSH\" -Name DefaultShell " +
```

Comment:
This changes the default shell for SSH to PowerShell. Shouldn't you be replacing the default `Command Prompt` with `PowerShell`? IOW, does changing the default prompt to PowerShell translate into it being the default in SSH or is that a separate step? Can I can have the Windows instance default to `PowerShell` but still use `cmd` in SSH?

Reply:
I'm not sure what you mean. Are you saying if a user is physically accessing the VM? In that case if they had `Windows Server with Desktop Experience`  they would either open the CMD or PowerShell executables. If using a different remote management tool the shell would need to be configured for that tool.